.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: python
+python: '3.7'
+services: docker
+script: docker run --rm -tv $(pwd):/src -w /src ghdl/vunit:gcc-master python3 /src/run.py
```

Comment:
Do we need code coverage? Otherwise, we'd better use `ghdl/vunit:llvm-master`. At the same time, I don't know if it is desirable to use `master` version of VUnit instead of the latest stable version published to PyPI (i.e. `ghdl/vunit:llvm`).

Reply:
I thought we might have that as a quality measure for our compliance tests but we can wait with that until that is done. You're right, not using master is a bit more safe.

Reply:
Seems like vunit:llvm is failing while vunit:llvm-master is not. Uisng the latter until that issue is resolved.

Reply:
> I thought we might have that as a quality measure for our compliance tests

If this is the case, I don't mind using gcc directly. I thought we were not going to use coverage.

> but we can wait with that until that is done.

The main difference is that gcc-master (340MB) is 2.7 times larger than llvm (126MB): https://hub.docker.com/r/ghdl/vunit/tags

> Seems like vunit:llvm is failing while vunit:llvm-master is not. Using the latter until that issue is resolved.

I think the approach should be to fix VUnit and release a new version, before merging this PR. Nonetheless, we can keep `-master` meanwhile to test other features of the PR.

Reply:
Turns out to be trivial. My solution builds on the `--minimal` option which wasn't part of 4.1.0. So, it's nothing wrong with 4.1.0, it's only that my solution depends on HEAD.

Reply:
Builds with latest VUnit release now. @1138-4EB Ready to merge